frontend/javascripts/oxalis/view/components/editable_text_label.tsx

```diff
@@ -163,31 +163,27 @@ class EditableTextLabel extends React.PureComponent<EditableTextLabelProp, State
     };
 
     if (this.state.isEditing) {
-      return (
-        <span style={{ display: "inline-flex", alignItems: "center" }}>
-          {this.props.rows === 1 ? (
-            <React.Fragment>
-              <Input {...inputComponentProps} onBlur={() => this.handleOnChange} />
-              <FastTooltip key="save" title={`Save ${this.props.label}`} placement="bottom">
-                <CheckOutlined
-                  style={iconStyle}
-                  onClick={(evt) => {
-                    evt.stopPropagation();
-                    this.handleOnChange();
-                  }}
-                />
-              </FastTooltip>
-            </React.Fragment>
-          ) : (
-            <MarkdownModal
-              source={this.state.value}
-              isOpen={this.state.isEditing}
-              onChange={this.handleInputChange}
-              onOk={this.handleOnChange}
-              label={this.props.label}
+      return this.props.rows === 1 ? (
+        <Space.Compact block>
```

Comment:
replace outer `<span>` and `<React.Fragement>` with `<Space>`. The rest is identical.